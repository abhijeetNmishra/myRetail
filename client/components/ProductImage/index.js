import React from 'react';

const ProductImage = (props) => {
  const image = props.images;
  const primaryImage = image.PrimaryImage[0].image;
  return (
    <div className="row image-container">
      <img id="primaryImage" src={primaryImage} alt="primary" />
    </div>
  );
};

ProductImage.propTypes = {
  images: React.PropTypes.object
};

export default ProductImage;
