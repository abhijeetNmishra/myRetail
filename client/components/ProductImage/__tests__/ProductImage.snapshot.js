/* eslint-disable */
import React from 'react';
import renderer from 'react-test-renderer';
import ProductImage from '../index';

test('Layout renders correctly', () => {
  const images = {
   "AlternateImages": [
    {
     "image": "http:\/\/target.scene7.com\/is\/image\/Target\/14263758_Alt01"
    },
    {
     "image": "http:\/\/target.scene7.com\/is\/image\/Target\/14263758_Alt02"
    },
    {
     "image": "http:\/\/target.scene7.com\/is\/image\/Target\/14263758_Alt03"
    },
    {
     "image": "http:\/\/target.scene7.com\/is\/image\/Target\/14263758_Alt04"
    },
    {
     "image": "http:\/\/target.scene7.com\/is\/image\/Target\/14263758_Alt05"
    },
    {
     "image": "http:\/\/target.scene7.com\/is\/image\/Target\/14263758_Alt06"
    },
    {
     "image": "http:\/\/target.scene7.com\/is\/image\/Target\/14263758_Alt07"
    }
   ],
   "PrimaryImage": [
    {
     "image": "http:\/\/target.scene7.com\/is\/image\/Target\/14263758"
    }
   ],

   "imageCount": "8",
   "source": "internal"
  }

  const productImage = renderer.create(
    <ProductImage images={images}></ProductImage>
  ).toJSON();
  expect(productImage).toMatchSnapshot();
});
