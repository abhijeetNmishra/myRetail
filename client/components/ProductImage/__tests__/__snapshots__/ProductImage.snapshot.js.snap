exports[`test Layout renders correctly 1`] = `
<div
  className="row h-text-center">
  <img
    src="http://target.scene7.com/is/image/Target/14263758" />
</div>
`;
